scripts/pylib/twister/twisterlib.py

```diff
@@ -2715,6 +2720,12 @@ def run(self):
             if instance.handler.type_str == "device":
                 instance.handler.suite = self.suite
 
+            if(instance.platform.type == "native"):
```

Comment:
Why just in native?

Reply:
The seed argument is only valid when configured to use `fake_entropy_native_posix.c`.  Passing seed for any other selected config will cause an error.  The random number generator may use a different entropy module for random numbers.

Reply:
native_posix is not the only "native" target type. Upstream nrf52_bsim is also (and others downstream). nrf52_bsim supports --random_seed (or --rs), but not --seed

Reply:
Thanks - will update to reflect `native_posix`